module #3. REST API Advanced/src/main/java/com/epam/esm/ApplicationRunner.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@EnableJpaRepositories
+@EnableTransactionManagement
+@EnableJpaAuditing
+@SpringBootApplication
+public class ApplicationRunner {
```

Comment:
Its better to use some name like GiftCertificateApplication with "Application" suffix. And i believe that you can ommit @EnableTransactionManagement and @EnableJpaRepositories as they are included into spring boot starter jpa autoconfiguration.